migrations/migrationHelpers.js

```diff
@@ -117,11 +122,40 @@ const getdeployedContractInstance = async (
   return { Contract, contractInstance };
 };
 
+const source = 'https://raw.githubusercontent.com/razor-network/datasources/master';
```

Comment:
String consts need to be in uppercase. const SOURCE = '...'

Reply:
resolved 57925d7